packages/office-ui-fabric-react/src/components/Persona/PersonaCoin.tsx

```diff
@@ -149,7 +149,7 @@ export class PersonaCoin extends React.Component<IPersonaProps, IPersonaState> {
               shouldStartVisible={ imageShouldStartVisible }
               onLoadingStateChange={ this._onPhotoLoadingStateChange }
             />
-            <PersonaPresence { ...this.props } />
+            { !coinSize && <PersonaPresence { ...this.props } /> }
```

Comment:
I don't quite think this is predictable. A customer who has a 20px coin, and needs a 26px would suddenly have the presence disappear. That's weird right?

Reply:
Matt had some great points from a design perspective about why they want to keep this behavior. I asked him to post here so we can continue the conversation.

Reply:
Matt?